src/browser/app/App.react.js

```diff
@@ -17,14 +17,20 @@ import {connect} from 'react-redux';
 export default class App extends Component {
 
   static propTypes = {
+    actions: PropTypes.object.isRequired,
     children: PropTypes.object.isRequired,
     location: PropTypes.object.isRequired,
     msg: PropTypes.object.isRequired,
-    users: PropTypes.object.isRequired
+    persistence: PropTypes.object.isRequired
```

Comment:
As I said, I think it was less complicated to touch the persistence store (cookies) only in actions and not sync it with appState.